homeassistant/components/hydrawise/sensor.py

```diff
@@ -56,18 +58,24 @@ def setup_platform(
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Set up a sensor for a Hydrawise device."""
-    coordinator: HydrawiseDataUpdateCoordinator = hass.data[DOMAIN]
-    hydrawise: LegacyHydrawise = coordinator.api
-    monitored_conditions = config[CONF_MONITORED_CONDITIONS]
+    return
```

Comment:
Probably make a comment here that the import is triggered from `__init__.py` which the user needed to setup from anyhow.

Reply:
Added a comment. Hopefully it's what you meant?